libs/pageserver_api/src/models.rs

```diff
@@ -356,6 +465,57 @@ pub struct TenantConfig {
     pub wal_receiver_protocol_override: Option<PostgresClientProtocol>,
 }
 
+impl TenantConfig {
+    pub fn apply_patch(mut self, patch: TenantConfigPatch) -> TenantConfig {
```

Comment:
Please rewrite this function to use exhaustive destructuring of `self`.
That way, we'll get compiler errors whenever changing `struct TenantConfig`.

Reply:
Done in https://github.com/neondatabase/neon/pull/10020/commits/c1115bf3a7c62d8c042548ef2a11ada0295b1a44. Thanks! Good suggestion.